coaster/utils.py

```diff
@@ -421,6 +421,45 @@ def isoweek_datetime(year, week, timezone='UTC', naive=False):
         return dt
 
 
+def midnight_to_utc(dt, timezone=None, naive=False):
+    """
+    Returns a UTC datetime matching the midnight for the given date or datetime.
```

Comment:
One last quibble. There should be a blank line between explanation and doctest. Good to merge after that.

Oh, also a line in `CHANGES.rst` about adding this new function.